src/test.ts

```diff
@@ -1,8 +1,8 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
-
+// tslint:disable:ordered-imports
```

Comment:
Developers might find the reason of disabling imports not apparent.
Could you add a comment about the reason this is necessary?

Reply:
Thanks for the suggestion! Just committed the new changes.